request.js

```diff
@@ -116,15 +116,15 @@
         return callbacks;
     };
 
-    exports['get'] = function (url, query) {
+    exports.get = function (url, query) {
```

Comment:
Because now you have 3 methods returned with `exports[]` and only one with `exports.` and that's ugly